src/pages/slash-graphql/advanced-queries.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: "Advanced Queries with GraphQL+-"
+redirect_from:
+  - /slash-graphql/advanced-queries/
+---
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+*These are draft docs for Slash GraphQL, which is currently in beta*
+
+Slash GraphQL also supports running advanced queries with `GraphQL+-`, a query language that is unique to Dgraph. GraphQL+- should be used by advanced users who wish to make queries and mutations using existing Dgraph client libraries, either via the HTTP or gRPC endpoints. You can learn more about existing client libraries by following this [documentation](https://dgraph.io/docs/clients/).
+
+If you are getting started with Slash GraphQL, you might want to consider using our [GraphQL APIs](https://graphql.dgraph.io/doc/) instead. It will get you quickly started on the basics of using Slash GraphQL before you go into advanced topics. 
+
+Do note that Slash GraphQL does not allow you to change the schema or create new predicates via GraphQL+-. You will not be able to access the `/alter` endpoint. You can add your schema via Slash GraphQL (either UI or API) and then add your mutations according to your Schema using GraphQL+-.
+
+## Authentication and Endpoint
+
+It is important to note that all APIs must be authenticated by an API key from the settings page. This can be accessed via Slash GraphQL UI, navigate to the [settings](https://slash.dgraph.io/_/settings) tab and you will see an option to create a new API key. Make sure to keep your API key safe, it will not be accessible once you navigate away from the page. Your API key will never expire so you don't have to worry about changing your API key. 
+
+### HTTP
+You will be able to find your base URL (GraphQL Endpoint) on your [Dashboard](https://slash.dgraph.io/_/dashboard) after your deployment is up. You will have access to `/mutate`, `/query` and `/commit` endpoints that you can use as per your convenience. 
```

Comment:
give an example. Don't say "as per your convenience". Instead say 'to access the data" or something like that. Also, link to the relevant docs

Reply:
Also, give an example of the endpoint. Copy the text from the other PRt